config/manifests/4.10/aws-efs-csi-driver-operator.clusterserviceversion.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
     capabilities: Full Lifecycle
     containerImage: quay.io/openshift/origin-aws-efs-csi-driver-operator:latest
     support: Red Hat
+    operators.openshift.io/infrastructure-features: '["disconnected"]'
```

Comment:
I think we should also add `proxy-aware` to this list. What do you think? @jsafrane @duanwei33

Reply:
Oh, and we definitely need to add `csi`! https://github.com/openshift/openshift-docs/blob/main/modules/osdk-csv-manual-annotations.adoc

Reply:
I am not sure about the proxy. Sure, API calls to AWS can (and should) be proxied, however, the CSI driver / efs-utils directly mount NFS from somewhere inside AWS, which will not be proxied.

I added `"csi"`.